server/model/schema.sql

```diff
@@ -1,6 +1,6 @@
--- DROP TABLE sellerAccount CASCADE;
--- DROP TABLE category CASCADE;
--- DROP TABLE adListing CASCADE;
+DROP TABLE sellerAccount CASCADE;
+DROP TABLE category CASCADE;
+DROP TABLE adListing CASCADE;
```

Comment:
I'm guessing this was added again due to the categories being added each time it is deployed instead of just once? We should be careful about dropping the tables on start up. Now that we have Ads being added to the DB too we don't want them to be dropped! If my assumption is correct what I would suggest instead is that in `data.sql` we change the insert statements to instead be something like 'insert if not exists'. @PeterArmstrong79 this may be a task that you would like to pick up to get more SQL exp? Here is an example of what I mean:
https://stackoverflow.com/questions/4069718/postgres-insert-if-does-not-exist-already

Reply:
@LiamGve Yes, I had a chat with @PeterArmstrong79 regarding this. I think he will solve this problem later. So for now for this PR what do you think is better: left them as they are or still comment them out?

Reply:
Will have a look

Reply:
I think still comment them out for now to avoid confusion

Reply:
@LiamGve Done

Reply:
@zhkvivan approved!